modules/ossm-remove-cleanup.adoc

```diff
@@ -8,11 +8,6 @@
 
 Follow this procedure to manually remove resources left behind after removing the {ProductName} Operator using the OpenShift console.
```

Comment:
s/using/by using/

ISG "Grammar" heading, example on top of p24 of _my version_.